src/etos_lib/logging/logger.py

```diff
@@ -150,7 +161,41 @@ def setup_rabbitmq_logging(log_filter):
     root_logger.addHandler(rabbit_handler)
 
 
-def setup_logging(application, version, environment, config_file=DEFAULT_CONFIG):
+def setup_otel_logging(
+    log_filter: EtosFilter,
+    resource: Resource,
+    log_level: int = logging.INFO,
+) -> None:
+    """Set up OpenTelemetry logging signals.
+
+    :param log_filter: Logfilter to add to OpenTelemetry handler.
+    :type log_filter: :obj:`EtosFilter`
```

Comment:
Nit: The :types: here in docstring are, in my opinion, unnecessary when using type hints.

Reply:
I thought about this, and I left it there because I imagined there would be some auto generated docs visualization that might break. However, looking into it now such fancy docs tool does not exist, so I guess it can be removed.